Essentials/src/com/earth2me/essentials/User.java

```diff
@@ -584,8 +584,15 @@ public void updateActivity(final boolean broadcast) {
             if (broadcast && !isHidden()) {
                 setDisplayNick();
                 final String msg = tl("userIsNotAway", getDisplayName());
+                final String selfmsg = tl("userIsNotAwaySelf");
                 if (!msg.isEmpty()) {
```

Comment:
Okay so pretend you are reading the message I wrote below about msg.isEmpty first....

So yeah this confirms my suspicion - if that message is emptied, the message isn't broadcast. There should be some more consistent behavior - currently if msg is empty, the selfmsg won't be sent even if defined and broadcastAfkMessage is false (as an owner, I would expect selfmsg to be sent to the user)

Reply:
Ah, I missed that - thanks! I'm now checking each message separately before attempting to send them to the player.